core/driver/sources/src/test/scala/org/virtuslab/ideprobe/dependencies/SingleRunFixtureTest.scala

```diff
@@ -87,4 +89,17 @@ final class SingleRunFixtureTest extends IdeProbeFixture with WorkspaceFixture w
       workspacesNotDeleted.isEmpty
     )
   }
+
+  @Test
+  def copiesDiagnosticsDataIntoConfiguredDirectoryBeforeCleanup(): Unit = {
+    val config = Config.fromString("""probe.paths.diagnostics = "/tmp/ide-probe-test"""")
+    val fixtureFromConfig = IntelliJFixture.fromConfig(config)
+    val diagnosticsFixture = new SingleRunIntelliJ(fixtureFromConfig)
+
+    diagnosticsFixture { runningIntellijFixture =>
+      // nothing special to be done here
+    }
+
+    assertTrue(Paths.get(s"/tmp/ide-probe-test").directChildren().nonEmpty)
```

Comment:
Maybe check for `logs` child dir explicitly?

Reply:
Changed tests logic - please take a look